.github/workflows/pylint.yml

```diff
@@ -0,0 +1,30 @@
+name: Lint Code with Pylint
+
+on:  
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.12"
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+
```

Comment:
Pylint needs to know which Django settings module to use, otherwise, it might fail when checking Django-specific code. Set the `DJANGO_SETTINGS_MODULE` environment variable before running pylint.